packages/dds/tree/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -90,21 +83,22 @@ const testCases: (readonly [string, FieldKey])[] = [
 describe("editable-tree: editing", () => {
     it("assert set primitive value using assignment", async () => {
         const [, trees] = await createSharedTrees(fullSchemaData, [personData]);
-        const person = trees[0].root as PersonType;
+        const person = trees[0].root as Person;
         const nameNode = person[getField](brand("name")).getNode(0);
         const ageNode = person[getField](brand("age")).getNode(0);
-        const phonesField = person.address.phones;
-        assert(isEditableField(phonesField));
 
         assert.throws(
-            () => (person.friends[valueSymbol] = { kate: "kate" }),
+            () => {
+                assert(person.friends !== undefined);
+                person.friends[valueSymbol] = { kate: "kate" };
+            },
             (e) => validateAssertionError(e, "The value is not primitive"),
             "Expected exception was not thrown",
         );
         assert.throws(
             () => {
```

Comment:
try and keep code in a scope which catches exceptions (like `assert.throws`) minimal. If this will build with this assert moved to before the `assert.throws` that would be preferred. Same for the above case as well.

Reply:
I agree. Unfortunately, I have tried different ways of writing it down, but this is the only way allowed.